appveyor.yml

```diff
@@ -1,22 +1,33 @@
 version: '{build}'
-image: Visual Studio 2019
+
+image:
+  - Visual Studio 2019
+  - Ubuntu1804
 
 pull_requests:
   do_not_increment_build_number: true
 
-build_script:
-- cmd: build.cmd
+for:
+- matrix:
+    only:
+      - image: Visual Studio 2019
+  build_script:
+    - cmd: build.cmd
+  deploy_script:
+    - cmd: tools\deploy.cmd
+- matrix:
+    only:
+      - image: Ubuntu1804
+  build_script:
+    - sh: ./build.sh
 
 on_failure:
   - ps: $root = (Resolve-Path artifacts); [IO.Directory]::GetFiles($root.Path, '*.*', 'AllDirectories') | % { Push-AppveyorArtifact $_ -FileName $_.Substring($root.Path.Length + 1) -DeploymentName failed-build }

```

Comment:
I'm thinking that at best, this will not fire on ubuntu. At worst, it'll cause a more confusing error. What think you?

Reply:
> I'm thinking that at best, this will not fire on ubuntu

I'm not so sure... The Ubuntu images come with Powershell Core installed. But maybe we need to switch to `pwsh` instead of `ps`.

Reply:
Actually, it works with `ps`. See [this failed build](https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/builds/28409128/job/03xbdnjcylmfutv5)